app/models/custom_form.py

```diff
@@ -133,9 +133,9 @@
         "instagram": "Instagram",
         "gender": "Gender",
         "ageGroup": "Age Group",
-        "acceptVideoRecording": "Accept Video Recording",
-        "acceptShareDetails": "Accept Share Details",
-        "acceptReceiveEmails": "Accept Receive Emails",
+        "acceptVideoRecording": "Photo & video & text consent",
```

Comment:
Please make sure that this is actually what is needed, or else another migration will be needed

Reply:
In the issues @mariobehling mentioned these only

Reply:
Yes, I clarified that now. Actually sometimes is differs for events, but this seems to be suitable for the meantime until we can have the tick box for custom questions.